01week/rockPaperScissors.js

```diff
@@ -1,51 +1,63 @@
-'use strict';
+"use strict";
 
-const assert = require('assert');
-const readline = require('readline');
+const assert = require("assert");
+const readline = require("readline");
 const rl = readline.createInterface({
   input: process.stdin,
   output: process.stdout
 });
 
-
 function rockPaperScissors(hand1, hand2) {
```

Comment:
While the game is played correctly, this program will not pass the tests. Can you see why?